samples/maps/demo/basic-map/demo.js

```diff
@@ -0,0 +1,63 @@
+(async () => {
+
+    const topology = await fetch(
+        'https://code.highcharts.com/mapdata/custom/world.topo.json'
+    ).then(response => response.json());
+
+    Highcharts.mapChart('container', {
+        chart: {
+            map: topology
+        },
+
+        title: {
+            text: 'Life expectancy in the world (2021)'
```

Comment:
Small pirk: The "in the world" part reads a bit weird to me. Maybe "World life expectancy" or "Life expectancy by country"?

Reply:
The last one sounds good to me!